common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1091,6 +1091,12 @@
     ],
     "sqlState" : "KD010"
   },
+  "DATA_SOURCE_UNSUPPORTED_DML_OPERATION" : {
+    "message" : [
+      "Data source '<provider>' does not support requested DML operation: <sqlOperation>"
+    ],
+    "sqlState" : "KD011"
```

Comment:
You need to propose a KD011, it seems new.

Reply:
Done, thanks